R/config-globalConstant.R

```diff
@@ -7,11 +7,11 @@ NUM_PARAMETERS_INDEPENDENT <- 4
 NUM_PARAMETERS_SARMANOV <- 5
 
 ### path of share library
-if(.Platform$OS.type=="unix"){
-  SHARED_LIBRARY_PATH <-file.path(find.package("mmeta"),"libs","mmeta.so")
-} else{
-  if(.Platform$r_arch=="i386") 
-    SHARED_LIBRARY_PATH <- file.path(find.package("mmeta"),"libs","i386","mmeta.dll")
-  if(.Platform$r_arch=="x64")  
-    SHARED_LIBRARY_PATH <- file.path(find.package("mmeta"),"libs","x64","mmeta.dll")  
-}
+ # if(.Platform$OS.type=="unix"){
+ #  SHARED_LIBRARY_PATH <-file.path(find.package("mmeta"),"libs","mmeta.so")
+ # } else{
+ #  if(.Platform$r_arch=="i386")
+ #    SHARED_LIBRARY_PATH <- file.path(find.package("mmeta"),"libs","i386","mmeta.dll")
```

Comment:
Please delete those lines of codes which are commented out.

Reply:
The coments have been deleted.